src/ase/cw/utlities/ServerStatusEnum.java

```diff
@@ -2,6 +2,6 @@
 
 public class ServerStatusEnum {
 
-    public enum ServerStatus {FREE, BUSY, PAUSED,STOPPED}
+    public enum ServerStatus {FREE, BUSY, PAUSED,GOING_TO_PAUSED,GOING_TO_STOP,STOPPED}
```

Comment:
please use more consistent naming, i.e. TO_BE_PAUSED/TO_BE_STOPPED or GOING_TO_PAUSED/GOING_TO_STOPPED

Reply:
fixed

> I think you haven't merged my last night's changes into your branch fully so you're basically revoking some of them (I am mostly referring to the changes to ServerFrame). Please also create an issue item for these changes - this generally needs to be done before you start working on fixes or features so that others know what you are working on and there is no duplication of effort and conflicts as a result (I fixed #138 with #146 )

Yes, we worked in parallel yesterday, so i got some merge conflicts, and I obviously deleted some of your code lines. I mostly only reduced the complexity of the controller class. I fixed #138 and #147 without purpose^^ .